fvm/state/state_test.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/onflow/flow-go/fvm/utils"
 )
 
+func createByteArray(size int) []byte {
```

Comment:
what is this function for?

Reply:
it's copied from accounts_test.go.  we need to duplicated it here since it's also used by these tests